ecole-doctorale-de-sciences-po-author-date-french.csl

```diff
@@ -0,0 +1,440 @@
+      </else-if>
+      <else-if type="article" match="any">
+      <group>
+          <text variable="title" text-case="capitalize-first" font-style="italic" suffix=". "/>
+        </group>
+      </else-if>
+      <else>
+        <text variable="title" quotes="true"/>
+      </else>
+    </choose>
+  </macro>
+<macro name="pub-place">
+    <choose>
+      <if type="bill book chapter entry-dictionary entry-encyclopedia thesis legal_case manuscript map paper-conference song" match="any">
+        <choose>
+          <if variable="publisher-place" match="any">
+            <text variable="publisher-place" suffix=" : "/>
+          </if>
+          <else>
+            <text value="[s.l.]"/>
+          </else>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+ <macro name="publisher">
+    <choose>
+      <if type="bill book chapter entry-dictionary entry-encyclopedia thesis legal_case manuscript map paper-conference report song" match="any">
+        <choose>
+          <if variable="publisher" match="any">
+            <text variable="publisher"/>
+          </if>
+          <else>
+            <text value="[s.n.]"/>
+          </else>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+<macro name="year">
+<choose>
+      <if type="article article-journal article-newspaper article-magazine bill book broadcast chapter entry-dictionary entry-encyclopedia graphic legal_case manuscript  map motion_picture paper-conference patent post post-weblog personal_communication report song speech thesis webpage" match="any">
+<group delimiter=". ">
+          <date variable="issued">
+            <date-part name="year"/>
+          </date>
+</group>
+</if>
+    </choose>
+  </macro>
+  <macro name="yearpage">
+    <choose>
+      <if type="article bill book graphic legal_case map motion_picture paper-conference manuscript report song thesis " match="any">
+<group delimiter=". ">
+          <date variable="issued">
+            <date-part name="year"/>
+          </date>
+          <group>
+            <text term="volume" form="short" suffix="."/>
+            <text variable="number-of-volumes" prefix=". " suffix="/"/>
+            <text variable="volume"/>
+          </group>
+          <choose>
+            <if variable="locator" match="any">
+              <group delimiter="&#8239;">
+                <label variable="locator" form="short"/>
+                <text variable="locator"/>
+              </group>
+            </if>
+            <else-if variable="locator" match="none">
+              <text variable="number-of-pages" suffix="&#160;p"/>
+            </else-if>
+          </choose>
+        </group>
+      </if>
+      <else-if type="chapter entry-dictionary entry-encyclopedia" match="any">
+        <group delimiter=" " font-style="normal">
+          <date variable="issued">
+            <date-part name="year" suffix=", "/>
+          </date>
+          <group>
+            <text term="volume" form="short" suffix="."/>
```

Comment:
the term already has a . -- you're setting it twice more, once in the suffix, once in the prefix. Remove.

Reply:
ok